apps/testing-nested/src/app/child.component.cy.ts

```diff
@@ -4,15 +4,46 @@ describe('ChildComponent', () => {
   const setup = () => {
     cy.mount(ChildComponent);
   };
+
   describe('When typing nothing and clicking on Validate', () => {
     test('Then show "Title is required" error message and no http request has been sent', async () => {
       setup();
+
+      cy.window()
+        .its('console')
+        .then((console) => {
+          cy.spy(console, 'log').as('log');
+        });
```

Comment:
suggestion: Listening to console.log is not very clean. A better way is to mock your service and listen to the function call.